client/lib/route/page-notifier.js

```diff
@@ -1,23 +1,34 @@
 /*eslint no-multi-spaces: 0*/
 // node dependency
 var EventEmitter   = require( 'events' ).EventEmitter;
+var ipc = require( 'electron' ).ipcRenderer;
+var includes = require( 'lodash' ).includes;
 
 // internal dependency
-var page           = require( 'page' );
-var Route          = page.Route;
+var page		   = require( 'page' );
+var Route		  = page.Route;
 
 // routing state and middleware
-var location       = new EventEmitter();
+var location	   = new EventEmitter();
 var currentContext = null;
-var route          = new Route( '*' );
+var commandLine	= {};
+var route		  = new Route( '*' );
 
 // Inject our route at the front so we are always called
 page.callbacks.unshift( route.middleware( function( context, next ) {
+	context.commandLine = commandLine;
```

Comment:
@johngodley : No, it is not sensitive to the order and the command line will be read only after it is starting to be rad from.
With my new commit the app can also be running initialized elsewhere and information will still be passed.
The code runs only after the page is loaded.